app/src/main/java/com/tpp/theperiodpurse/ui/setting/SettingScreenLayout.kt

```diff
@@ -75,6 +80,18 @@ fun SettingScreenLayout(
         }
     )
 
+    val hasNotificationPermission by remember {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            mutableStateOf(
+                ContextCompat.checkSelfPermission(
+                    context,
+                    Manifest.permission.POST_NOTIFICATIONS
+                ) == PackageManager.PERMISSION_GRANTED
+            )
+        } else mutableStateOf(true)
+    }
+//    appViewModel.setAllowReminders(hasNotificationPermission)
+
```

Comment:
Is this exactly the same as above?

Reply:
Sorry! I was supposed to change the permission. I've fixed it now